style.scss

```diff
@@ -83,6 +85,9 @@
   overflow-y: scroll;
```

Comment:
Can we change height to percentage(e.g. 80%) rather than fixed pixal? This is related to what I brought up on slack

Reply:
sorry which height?

Reply:
Also, I've deleted .DS_Store

Reply:
Oh I know what you mean now

Reply:
Sorry I meant the height for interactions, it is set to 600px

Reply:
Line [here](https://github.com/shanliyu/chocopy-wasm-compiler/blob/4810b192f773d5287993bba08ab9462f30b0cea6/style.scss#L148)

Reply:
It seems impossible to let height be the percentage of the screen height in css. So I still used fixed pixel, but added a breakpoint at height=900px for now. If it still doesn't look good on your device, we may further adjust it.

Reply:
Got it. Thanks for looking into this!